platform/solr/solr-banana-ingester/pom.xml

```diff
@@ -0,0 +1,177 @@
+            <groupId>org.noggit</groupId>
+            <artifactId>noggit</artifactId>
+            <version>0.5</version>
+        </dependency>
+        <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
+        </dependency>
+    </dependencies>
+
+    <!--TODO: could also override default.json to create custom default dashboard-->
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <configuration>
+                    <instructions>
+                        <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+                        <Embed-Dependency>
+                            guava,
+                            Saxon-HE,
+                            solr-factory,
+                            solr-core,
+                            solr-solrj,
+                            httpclient,
+                            httpcore,
+                            httpmime,
+                            commons-lang,
+                            org.apache.commons,
+                            saxon,
+                            noggit
+                        </Embed-Dependency>
+                        <Import-Package>
+                            javax.xml.parsers,
+                            org.xml.sax,
+                            org.xml.sax.helpers,
+                            org.xml.sax.ext,
+                            org.apache.commons.lang,
+                            org.apache.commons.logging,
+                            org.slf4j,
+                            org.codice.ddf.configuration,
+                            javax.net.ssl,
+                            javax.net,
+                            javax.security.auth.x500,
+                            javax.naming.ldap,
+                            javax.naming.directory,
+                            javax.naming
+                        </Import-Package>
+                    </instructions>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>org.jacoco</groupId>
+                <artifactId>jacoco-maven-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>default-check</id>
+                        <goals>
+                            <goal>check</goal>
+                        </goals>
+                        <configuration>
+                            <haltOnFailure>true</haltOnFailure>
+                            <rules>
+                                <rule>
+                                    <element>BUNDLE</element>
+                                    <limits>
+                                        <limit>
+                                            <counter>INSTRUCTION</counter>
+                                            <value>COVEREDRATIO</value>
```

Comment:
At the very least you should probably have an integration test that verifies that this can be deployed and that the banana context is available.